storage/remote/store.go

```diff
@@ -193,6 +236,7 @@ func (s *Store) LedgerByHeight(
 			return value, nil
 		}
 
+		log.Printf("fetching register: owner: %x, key: %x", []byte(id.Owner), []byte(id.Key))
```

Comment:
Is this development/debug code? Maybe remove if not needed

Reply:
oops. good catch